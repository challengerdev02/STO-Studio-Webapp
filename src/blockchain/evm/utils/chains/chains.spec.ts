import { cleanup } from '@testing-library/react';
import { Chains } from './index';

describe('Test Chains object', () => {
  afterEach(cleanup);

  const setup = () => {
    return {
      ...Chains,
    };
  };

  it('should test Chains', function () {
    const res = setup();

    expect(res).toBeTruthy();
    expect(res['1'].chainId).toBe(1);
    expect(res['1'].network).toBe('mainnet');
    expect(res['10'].network).toBe('optimism');
    expect(res['10'].chain).toBe('ETH');
    expect(res['4'].networkId).toBe(4);
    expect(res['4'].network).toBe('rinkeby');
  });

  it('should match with snapshot', () => {
    // @ts-ignore

    const { container } = setup();

    expect(container).toMatchSnapshot();
  });
});
