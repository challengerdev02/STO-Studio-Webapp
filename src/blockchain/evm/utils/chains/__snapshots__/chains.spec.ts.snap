// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Chains object should match with snapshot 1`] = `undefined`;
